resources/conf/config.toml.template

```diff
@@ -479,18 +479,18 @@ enabled = true
 [tracing]
   # Enable/Disable tracing in Choreo Connect
   enabled = false
-  # Type of tracer exporter (e.g: azure, jaeger, zipkin)
-  type = "jaeger"
+  # Type of tracer exporter (e.g: azure, zipkin). Use zipkin type for Jaeger as well.
+  type = "zipkin"
```

Comment:
starting from line 500 has zipkin related configuration example already

Reply:
We need to provide zipkin as the type for testing jaeger as well.